packages/kotti-ui/source/kotti-button/types.ts

```diff
@@ -0,0 +1,28 @@
+import { Yoco } from '@3yourmind/yoco'
+
+export namespace KottiButton {
+	export enum Type {
+		PRIMARY = 'primary',
+		SECONDARY = 'secondary',
+		TEXT = 'text',
+		DANGER = 'danger',
+	}
+
+	export enum Size {
+		SMALL = 'small',
+		MEDIUM = 'medium',
```

Comment:
I think this is a breaking change

Reply:
True, `"medium"` (and btw neither `"md"`) is a supported prop. Will remove

Reply:
I think it’s much better if we support `medium` and default to it instead of `null`. I’ll finalize this and merge it now